.github/workflows/packages-build-manager.yml

```diff
@@ -128,7 +128,8 @@ jobs:
           # Every call to this workflow will be internally managed as amd64, which is synonymous with x86_64
           echo "generate_package.sh -a amd64 --tag ${{ env.TAG }} --system ${{ inputs.system }} $FLAGS"
           bash generate_package.sh -a amd64 --tag ${{ env.TAG }} --system ${{ inputs.system }} $FLAGS
-          echo "PACKAGE_NAME=$(find /tmp -maxdepth 1 -type f -name *manager*.${{ inputs.system }} -exec basename {} 2>/dev/null \;)" | tee -a $GITHUB_ENV
+          echo "PACKAGE_NAME=$(find /tmp -maxdepth 1 -type f -name *manager_*.${{ inputs.system }} -exec basename {} 2>/dev/null \;)" | tee -a $GITHUB_ENV
+          echo "PACKAGE_SYMBOLS_NAME=$(find /tmp -maxdepth 1 -type f -name *manager-*.${{ inputs.system }} -exec basename {} 2>/dev/null \;)" | tee -a $GITHUB_ENV
```

Comment:
Please add a `is_stage=true` build to https://github.com/wazuh/wazuh/pull/23760#issuecomment-2159633274, we need to be sure that the production nomenclature generation works

Reply:
Done